c/tskit/tables.h

```diff
@@ -2816,6 +2868,8 @@ int tsk_table_sorter_free(struct _tsk_table_sorter_t *self);
 
 /** @} */
 
+int tsk_table_collection_check_integrity(
```

Comment:
Looks like this is incidental? The function should be delcared above, I think you can delete this.

Reply:
Thanks, I must have accidentally put this in during one of my rebasing adventures. Fixed and squashed now.